lib/rubocop/cop/sorbet/mutable_constant_sorbet_aware_behaviour.rb

```diff
@@ -12,13 +12,23 @@ def prepended(base)
             base.def_node_matcher(:t_let, <<~PATTERN)
               (send (const nil? :T) :let $_constant _type)
             PATTERN
+            # @!method t_type_alias?(node)
+            base.def_node_matcher(:t_type_alias?, <<~PATTERN)
+              (block (send (const {nil? cbase} :T) :type_alias ...) ...)
+            PATTERN
+            # @!method type_member?(node)
```

Comment:
Just FYI, I dropped this YARD tag here as required by rubocop, but we've had good results with this tapioca compiler: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/sorbet/tapioca/compilers/rubocop.rb